markdown_lab_rs.py

```
@@ -7,16 +7,28 @@
 
 logger = logging.getLogger(__name__)
 
+class OutputFormat(str, Enum):
+    """Output format for HTML conversion"""
+    MARKDOWN = "markdown"
+    JSON = "json"
+    XML = "xml"
+
 # Try to import the Rust extension
 try:
+    from .markdown_lab_rs import OutputFormat as RustOutputFormat
```

Ruff (0.8.2)

`.markdown_lab_rs.OutputFormat` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)